2_Operations/work-process.md

```diff
@@ -2,28 +2,24 @@
-   * 12:00 UTC for the whole company in the #operations channel huddle followed immediately by separate meetings for the projects in their own channel huddles.
+* Every **Monday** we have a [Monday Meeting ("MM")](https://github.com/teamniteo/handbook/blob/master/2_Operations/kanban.md#monday-meetings):
+   * 12:00 UTC for the whole company in the Niteo Zoom room followed immediately by separate meetings for the projects in their own Zoom channels.
 
 * Every weekday Niteans should check their [GitHub notifications](https://github.com/notifications) so that they don't miss things that involve them.
 
-* A [Developer Session](#developer-session) is scheduled when we have a fair amount of items to talk about and is typically held right after the Monday Meeting.
+* A [Developers Session](#developers-session) is scheduled when we have a fair amount of items to talk about and is typically held right after the Monday Meeting.
 
 Niteans are encouraged to take Fridays off for a long weekend. Same as any other vacation off, make sure that your backup Nitean is available to handle any potential issues (see [Vacation policy](https://github.com/teamniteo/handbook/blob/master/5_People/benefits.md#vacation)).
 
 ## Quarterly Review and Planning
 
 Every quarter we take time to review, reflect, and plan for the next quarter. There are three tasks for the Quarterly Review and Planning: Partners Meeting, Quarterly Review, and All Hands Meeting.
 
-## Developer Session
+## Developers Session
 
-We are a remote company and as such sharing of ideas and lessons learned is impaired by the fact that we don't have the time or space to do that effectively.
+We are a remote company and as such sharing of ideas and lessons learned is impaired by the fact that we don't have the time or space to do that effectively. In the long term, this is hurting everyone and as such the Developers Session is a way to discuss recent ideas, questions and novelties.
 
-In the long term, this is hurting everyone and as such the Developer Session is a way to discuss ideas and questions that might have arisen lately. Followed by a short lightning talk(s), which can be impromptu or declared beforehand so others can join.
-
-You can learn something from every single person in the world, no matter what their opinion on any topic is. We also believe that this is a big part of open source and our company, being able to learn and collaborate with people from all over the world who have a wide variety of different opinions on how to do things. On the other hand, a lightning talk can be a perfect way for testing how people react to a talk that you are going to have at next conference or just getting rid of that presentation fear in front of people.
-
-In order for things to not explode, the Session is limited to 60 minutes.
+In order to prevent excessive bike-shedding, Developers Session is limited to 60 minutes.
```

Comment:
```suggestion
In order to prevent excessive [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality), Developers Session is limited to 60 minutes.
```